src/WindowsTracing/KrabsTracer.cpp

```diff
@@ -65,10 +94,8 @@ void KrabsTracer::SetIsSystemProfilePrivilegeEnabled(bool value) {
 
 void KrabsTracer::SetupStackTracing() {
   // Set sampling frequency for ETW trace. Note that the session handle must be 0.
-  const double frequency = capture_options_.samples_per_second();
-  ORBIT_CHECK(frequency >= 0);
-  if (frequency == 0) return;
-  double period_ns = 1'000'000'000.0 / frequency;
+  ORBIT_CHECK(sampling_frequency_hz_ >= 0);
+  double period_ns = 1'000'000'000.0 / sampling_frequency_hz_;
```

Comment:
You are dividing by zero here since that is how the construct the tracer in ListModulesEtw. I'm not sure what this does to the TRACE_PROFILE_INTERVAL  below (casting inf to ULONG?).